freqtrade/resolvers/strategy_resolver.py

```diff
@@ -79,6 +79,8 @@ def load_strategy(config: Dict[str, Any] = None) -> IStrategy:
                       ("sell_profit_only",                False,       'ask_strategy'),
                       ("ignore_roi_if_buy_signal",        False,       'ask_strategy'),
                       ("disable_dataframe_checks",        False,       None),
+                      ("ignore_buying_expired_candle",    None,       'ask_strategy'),
+                      ("ignore_buying_expired_candle_after",  0,       'ask_strategy')
```

Comment:
the 3rd parameter (`'ask_strategy'` defines the sub-key from the configuration.

I don't mind having it in ask_strategy as it's really related to trade entering anyway - but then the documentation should document it there, too.